scipy/stats/_continuous_distns.py

```diff
@@ -8324,26 +8262,16 @@ def _log_gauss_mass(a, b):
     case_right = a > 0.5
```

Comment:
```suggestion
    case_right = a > 0
```
The intent was "if we're in the right tail", where the cdf is greater than 0.5. This corresponds with `a > 0`. Please commit this with the other suggestions below if you agree it's OK to fix that here in this PR.